docs/overview/policy-lifecycle.mdx

```diff
@@ -4,62 +4,84 @@ sidebar_label: Policy Lifecycle
-infrastructure-as-code - they are stored and versioned in a git repository.
+At the core of Aserto’s authorization model is an authorization policy, which we refer to simply as a **Policy**. Policies are authored in a textual language called [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/), defined as part of the [Open Policy Agent](https://www.openpolicyagent.org/) (OPA) project in the [Cloud Native Computing Foundation](https://www.cncf.io/).
+
+Policies are treated just like application code or infrastructure-as-code - they can be stored as code in a git repository, or as an OCIv2 image in a policy repository (like the Aserto Policy Container Registry or the [Open Policy Registry](https://www.openpolicyregistry.io/)).
 
 ## The policy lifecycle
 
-Policies are **authored** using a text / code editor, **built** into an image, **published** to a repository hosted on 
-a registry, **run** in an Authorizer, and **consumed** by an application.
+Policies are **authored** using a text / code editor, **built** into an image, **published** to a repository hosted on a registry, **run** in a policy instance, and **consumed** by an application.
 
 ![Policy as code](/policy-as-code.png)
 
 Aserto's control plane helps manage the policy lifecycle.
 
 ### Author
 
-Policies are comprised of a collection of `.rego` files and `.json` files, which provide static data that 
-can be part of the policy.
+Policies are comprised of a collection of `.rego` files and `.json` files, which provide static data that can be part of the policy.
+
+### Build, tag and publish Policy images
+
+The set of source files that comprise a policy are built into a **Policy Image** using a set of tools that are modeled after the `docker` toolset. You can build and tag a policy just like you build and tag a docker image.
 
-### Build 
+There are two paths for managing the lifecycle of a policy image:
 
-The set of source files that comprise a policy are built into a **Policy Image** using a set of tools that 
-are modeled after the `docker` toolset.  You can build and tag a policy just like you build and tag a docker image.
+1. Using the `policy` CLI to build, tag and push a policy image to a policy registry.
+2. Using GitHub, a **build action** automatically builds a new version of a policy image when the source repository is **tagged** with a new tag.
 
-:::note
-Currently, the Aserto control plane wires up directly to a git repository that contains policy files, and 
-does not expose the policy image directly. A GitHub **build action** automatically builds a new version of 
-a policy image when the source repository is **tagged** with a new tag:
-:::
+### `policy` CLI
+
+#### Login
 
 ```bash
-git commit -am 'new policy version'
-git tag v0.1.2 && git push --tags
+policy login -s <PATH-TO-REGISTRY> -u <USERNAME> -p <PASSWORD>
 ```
 
-### Publish 
+To connect to the Aserto container registry, use the path `registry.prod.aserto.com`.
```

Comment:
I think the "password" should be the registry API key, right? Should be explicit about this...